python/lsst/ts/m2com/mock_model.py

```diff
@@ -1120,3 +1121,134 @@ def handle_position_mirror(self, mirror_position_set_point):
                 sleep(self.telemetry_interval)
 
         return True
+
+    def enable_open_loop_max_limits(self, status):
+        """Enable the maximum limits of open-loop control.
+
+        Parameters
+        ----------
+        status : `bool`
+            True if allow the maximum limits of open-loop control. Otherwise,
+            False.
+
+        Returns
+        -------
+        result : `bool`
+            True if succeeds. Otherwise, False.
+        """
+
+        result = True
+
+        if (status is True) and (self.force_balance_system_status is True):
+            self.is_enabled_open_loop_max_limits = False
+            result = False
+        else:
+            self.is_enabled_open_loop_max_limits = status
+
+        return result
+
+    def reset_breakers(self, power_type):
+        """Reset the breakers.
+
+        Parameters
+        ----------
+        power_type : `PowerType`
+            Power type.
+
+        Returns
+        -------
+        result : `bool`
+            True if succeeds. Otherwise, False.
+        """
+
+        result = False
+
+        if (power_type == PowerType.Motor) and self.motor_power_on:
+            result = True
+        elif (power_type == PowerType.Communication) and self.communication_power_on:
+            result = True
+
+        return result
+
+    def get_digital_output(self):
+        """Get the value of digital output.
+
+        Returns
+        -------
+        digital_output : `int`
+            Value of the digital output.
+        """
+
+        digital_output = (
+            DigitalOutput.InterlockEnable.value
+            + DigitalOutput.ResetMotorBreakers.value
+            + DigitalOutput.ResetCommunicationBreakers.value
+        )
+
+        if self.communication_power_on:
+            digital_output += DigitalOutput.CommunicationPower.value
+
+        if self.motor_power_on:
+            digital_output += DigitalOutput.MotorPower.value
+
+        return digital_output
+
+    def get_digital_input(self):
+        """Get the value of digital input.
```

Comment:
Maybe add:

```
Get the value of digital inputs that represents the current state of the system.
```

Reply:
Fixed in the commit: [8e6d00e](https://github.com/lsst-ts/ts_m2com/pull/2/commits/8e6d00e84f176e5c1227e83f56e9a124aa1afd53). Thanks!